src/main.py

```diff
@@ -36,25 +39,59 @@ def transcribe_time_stamps(segments: list):
     return string
 
 
-def make_srt_subtitles(segments: list):
+def split_text_by_punctuation(text: str, max_length: int):
```

Comment:
Well... after working a little bit with this, I found that some phrases were way too long for a subtitle, so I added a max_char value that you can tweak to make the subtitles look better. The way it works is that if the limit is exceeded, it'll look for a punctuation mark, question mark, or exclamation point to break the sentence in two, and split the time proportionally.